enterprise-suite/gotests/tests/prometheus/prometheus_test.go

```diff
@@ -126,6 +126,8 @@ var _ = Describe("all:prometheus", func() {
 		// All targets should be reachable
 		Expect(prom.HasData("up{kubernetes_name != \"es-test-service-with-only-endpoints\"} == 1")).To(Succeed())
 		Expect(prom.HasData("up{kubernetes_name != \"es-test-service-with-only-endpoints\"} == 0")).ToNot(Succeed())
+		// None of the metrics should have kubernetes_namespace label
+		// Expect(prom.HasData("{kubernetes_namespace!=\"\"}")).ToNot(Succeed())
```

Comment:
Did you mean to leave that second line commented out?

Reply:
Oops fixed